pkg/addressbook/addressbook.go

```diff
@@ -51,12 +51,16 @@ func New(storer storage.StateStorer) Interface {
 	}
 }
 
-func (s *store) Get(overlay swarm.Address) (bzz.Address, error) {
+func (s *store) Get(overlay swarm.Address) (*bzz.Address, error) {
 	key := keyPrefix + overlay.String()
-	v := bzz.Address{}
+	v := &bzz.Address{}
 	err := s.store.Get(key, &v)
 	if err != nil {
-		return bzz.Address{}, err
+		if err == storage.ErrNotFound {
```

Comment:
i find this change strange. if we dont find an entry this is an error, returning a `nil,nil` here is something i have never seen anywhere in the codebase. can you please explain the incentive of silencing this error? this is an anti pattern and breaks the convention of checking for error when accessing persistent storage. now all components using address book (and there's a few) will have to implement specific logic in order to avoid a nil pointer exception (panic!).